packages/rich-text/src/component/index.native.js

```diff
@@ -294,7 +298,20 @@ export class RichText extends Component {
 		this.lastAztecEventType = 'content size change';
 	}
 
+	onKeyDown( event ) {
+		if ( event.defaultPrevented ) {
+			return;
+		}
+
+		this.handleDelete( event );
+		this.handleEnter( event );
+		this.handleMention( event );
```

Comment:
Just a reminder that before merging this we need to either add a `DEV` flag here or remove it from where we add the mention button to the format bar.